src/core/struct_tree.js

```diff
@@ -322,6 +355,11 @@ class StructTreePage {
             type: "object",
             id: kid.refObjId,
           });
+        } else if (kid.type === StructElementType.ANNOTATION) {
+          obj.children.push({
+            type: "object",
```

Comment:
Is this correct, and if so a comment would definitely be in order, or should the type perhaps be e.g. `"annotation"` instead?